src/client.cpp

```diff
@@ -1307,21 +1311,27 @@ void Client::sendPlayerPos()
 
 	u8 camera_fov    = map.getCameraFov();
 	u8 wanted_range  = map.getControl().wanted_range;
+	float pitch = myplayer->getPitch();
+	float yaw = myplayer->getYaw();
+	if (map.getCameraFrontView()) {
+		yaw += 180 * 100;
```

Comment:
The math here is not strictly necessary; as long as the new and old values differ it's sufficient, but this way it's identical to the transformations in writePlayerPos.